Makefile

```diff
@@ -256,51 +257,62 @@ LOCALBIN := $(shell pwd)/bin
 $(LOCALBIN):
 	mkdir -p $(LOCALBIN)
 
-## Tool Binaries
-KUSTOMIZE ?= $(LOCALBIN)/kustomize
-CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
-ENVTEST ?= $(LOCALBIN)/setup-envtest
-
 ## Tool Versions
-KUSTOMIZE_VERSION ?= v3.8.7
-CONTROLLER_TOOLS_VERSION ?= v0.16.3
+KUSTOMIZE_VERSION ?= v5.7.0
+CONTROLLER_TOOLS_VERSION ?= v0.18.0
+# Set the Operator SDK version to use.
+OPERATOR_SDK_VERSION ?= v1.40.0
+# Set the OPM version to use.
+OPM_VERSION ?= v1.56.0
```

Comment:
It would be great to have the version in sync with everest-catalog https://github.com/percona/everest-catalog/blob/ede8e9ecde35c8bca8c1b4b696d50e8d6e26e985/.github/workflows/checks.yml#L24-L32